module/zfs/Makefile.in

```diff
@@ -106,21 +106,18 @@ $(MODULE)-objs += zcp_global.o
 $(MODULE)-objs += zcp_iter.o
 $(MODULE)-objs += zcp_synctask.o
 $(MODULE)-objs += zfeature.o
-$(MODULE)-objs += zfs_acl.o
```

Comment:
Why are files being removed here?

Reply:
It's just been prefixed with the path.

Reply:
@allanjude this line was moved to another Makefile